packages/manager/modules/enterprise-cloud-database/src/create/choose-database/choose-database.constants.js

```diff
@@ -0,0 +1,10 @@
+import PG_SQL from '../../assets/postgreSQL.svg';
+import MARIA_DB from '../../assets/mariadb.svg';
+
+export const PG_SQL_ICON = PG_SQL;
```

Comment:
Interesting, I don't know if this is a good idea. I feel like the files are self sufficient